sdk/eidolon_ai_sdk/bin/agent_http_server.py

```diff
@@ -93,14 +94,22 @@ def parse_args():
         help="Disable anonymous metrics collection",
         default=False,
     )
+    parser.add_argument(
+        "--fail-on-bad-agent",
+        help="Fail the server if an agent fails to start",
+        default=False,
```

Comment:
Should we default this to true? I imagine I as a dev would expect it to fail to startup if an agent was misconfigured. Then the error message would give me a message saying that I can run with the flag to startup anyways.

Reply:
Lol -- I was thinking the other way around. Let's discuss in person